src/ameba/ast/visitors/implicit_return_visitor.cr

```diff
@@ -168,10 +160,7 @@ module Ameba::AST
     def visit(node : Crystal::FunDef) : Bool
       report_implicit_return(node)
 
-      incr_stack do
-        node.args.each &.accept(self)
-        node.body.try &.accept(self)
-      end
+      incr_stack { node.accept_children(self) }
```

Comment:
now it traverses `node.return_type` which was not the case before. Is this expected? 

see https://github.com/crystal-lang/crystal/blob/master/src/compiler/crystal/syntax/ast.cr#L1968C8-L1968C19

Reply:
I thought it's fine as IIUC it doesn't change much.

Reply:
It won't cause any issues since it's being marked as used by the visitor